src/main/java/com/iConomy/MoneyCommand.java

```diff
@@ -0,0 +1,223 @@
+			} catch (IOException | InvalidConfigurationException e) {
+				continue;
+			}
+
+			if (account.getName().contains("-")) {
+				uuid = account.getName().replace(".yml", "");
+			}
+
+			if (uuid != null) {
+				name = data.getString("lastAccountName", "");
+				try {
+					money = Double.parseDouble(data.getString("money", "0"));
+				} catch (NumberFormatException e) {
+					money = 0;
+				}
+				String actualName;
+				/*
+				 * Check for Town/Nation accounts.
+				 */
+				if (hasTowny) {
+					if (name.startsWith(essTownPrefix)) {
+						actualName = name.substring(essTownPrefix.length());
+						log.info("Import: Town account found: " + actualName);
+						name = townPrefix + actualName;
+
+					} else if (name.startsWith(essNationPrefix)) {
+						actualName = name.substring(essNationPrefix.length());
+						log.info("Import: Nation account found: " + actualName);
+						name = nationPrefix + actualName;
+
+					} else if (name.startsWith(essDebtPrefix)) {
+						actualName = name.substring(essDebtPrefix.length());
+						log.info("Import: Debt account found: " + actualName);
+						name = debtPrefix + actualName;
+					}
+				}
+			}
+
+			try {
+				if (iConomy.Accounts.exists(name)) {
+					if (iConomy.Accounts.get(name).getHoldings().balance() == money) {
+						continue;
+					} else
+						iConomy.Accounts.get(name).getHoldings().set(money);
+				} else {
+					iConomy.Accounts.create(name);
+					iConomy.Accounts.get(name).getHoldings().set(money);
+				}
+
+				if ((i > 0) && (i % 10 == 0)) {
+					log.info(i + " accounts read...");
+				}
+				i++;
+
+			} catch (Exception e) {
+				log.warning("Importer could not parse account for " + account.getName());
+			}
+		}
+
+		log.info(i + " accounts loaded.");
+		return true;
+	}
+
+	private final List<String> SUB_CMDS = Arrays.asList("?", "rank", "top", "pay", "grant", "set", "hide", "create",
+			"remove", "preset", "purge", "empty", "stats", "help", "?");
+	private final List<String> PLAYER_CMDS = Arrays.asList("rank", "pay", "grant", "set", "hide", "create", "remove",
+			"reset");
+	private final List<String> AMOUNT_CMDS = Arrays.asList("pay","grant","set") ;
+
+	@Nullable
+	@Override
+	public List<String> onTabComplete(@NotNull CommandSender sender, @NotNull Command command, @NotNull String alias,
+			@NotNull String[] args) {
+
+		String subCmdArg = args[0].toLowerCase(Locale.ROOT);
+
+		if (args.length == 1) {
+			return SUB_CMDS.stream().filter(s -> s.startsWith(subCmdArg)).collect(Collectors.toList());
+		} else if (args.length == 2) {
+			if (PLAYER_CMDS.contains(subCmdArg))	
+				return null;
+			if (subCmdArg.equals("top"))
+				return Arrays.asList("<amount>");
+		} else if (args.length == 3) {
+			if (AMOUNT_CMDS.contains(subCmdArg))
+				return Arrays.asList("<amount>");
+			if (subCmdArg.equals("hide"))
+				return Arrays.asList("true", "false");
+		} else if (args.length == 4 && subCmdArg.equals("grant")) {
+				return Arrays.asList("true", "false");
```

Comment:
This isn't correct, please test all of the tab completions against the help menu to confirm it is all correct, before I retest it.